pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -1781,8 +1781,8 @@ class WorldSessionActor extends Actor
         log.info(s"MountVehicleMsg: $player_guid mounts $obj_guid @ $seat_num")
         PlayerActionsToCancel()
         sendResponse(PlanetsideAttributeMessage(obj_guid, 0, obj.Health))
-        sendResponse(PlanetsideAttributeMessage(obj_guid, 68, 0)) //shield health
-        sendResponse(PlanetsideAttributeMessage(obj_guid, 113, 0)) //capacitor
+        sendResponse(PlanetsideAttributeMessage(obj_guid, 68, obj.Shields)) //shield health
+        sendResponse(PlanetsideAttributeMessage(obj_guid, 113, obj.Capacitor)) //capacitor
```

Comment:
Maybe you can also add this : 
if(obj.Definition.ObjectId == 60){  // PTS v3
          sendResponse(PlanetsideAttributeMessage(obj_guid, 45, scala.math.ceil((obj.Capacitor.toFloat / obj.Definition.MaximumCapacitor.toFloat) * 10).toInt))
        }

Reply:
Hmm, perhaps we should be splitting out NTU capacitor from EMP capacitor, since they use different `PSAM` values

Reply:
Resolved